tests/framework/AnalyticModels/optimizing/maljovec.py

```diff
@@ -0,0 +1,17 @@
+# Function designed by D. Maljovech to have sinusoidal minimum
```

Comment:
I see I failed the surname spelling too.  I can change it to anything, no problem.